src/containers/Sidebar/index.js

```diff
@@ -47,28 +49,82 @@ const Sidebar = (props) => {
 		return itemsList;
 	}, [navConfig])
 
+	function useWindowDimensions() {
+		const [windowDimensions, setWindowDimensions] = useState(
+			getWindowDimensions()
+		);
+
+		useEffect(() => {
```

Comment:
@Aringoaway The same as I wrote above about the rule of not calling useEffect in nested function. It should be always top level